spec/features/saml/saml_spec.rb

```diff
@@ -167,6 +167,20 @@
   end
 
   context 'visiting /api/saml/logout' do
+    context 'via SP-initiated logout when not logged in to IdP' do
+      let(:user) { create(:user, :signed_up) }
+
+      it 'does not raise an exception' do
+        request = OneLogin::RubySaml::Logoutrequest.new
+        settings = sp1_saml_settings
+        settings.name_identifier_value = SecureRandom.uuid
+
+        expect  do
+          visit request.create(settings)
+        end.to_not raise_error NoMethodError
```

Comment:
We don't want to specify a type of error, because if any other error occurs, the test will pass. If you look at the test log, you'll see that RSpec outputs a warning about this kind of test:

```
WARNING: Using `expect { }.not_to raise_error(SpecificErrorClass)` risks 
false positives, since literally any other error would cause the 
expectation to pass, including those raised by Ruby (e.g. 
NoMethodError, NameError and ArgumentError), meaning the code you 
are intending to test may not even get reached. Instead consider using 
`expect {}.not_to raise_error`.
```